test/e2e/run.sh

```diff
@@ -61,18 +62,26 @@ for i in "$@"; do
     dev=yes
     shift # past argument=value
     ;;
+  --cleanup)
+    cleanup=yes
+    shift # past argument=value
+    ;;
```

Comment:
I added this so I could manually trigger a cleanup, because there were often services left running in the background, which caused problems for me (ports in use, etc.)

Reply:
👍